index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Toolbar -->
+    <nav class="toolbar">
+        <button type="button"><img src="logo/Icon - Menu.png" alt=""></b>
+    </nav>
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure that the opening `button` tag matches the closing one to avoid unnecessary errors and bad code format.
```suggestion
    <nav class="toolbar">
        <button type="button"><img src="logo/Icon - Menu.png" alt=""></button>
    </nav>
```